build/build/paths.yaml

```diff
@@ -1,5 +1,5 @@
 # This file is used to generate `target/<profile>/build/enso-build-<hash>/out/paths.rs`.
-# Generation logic is in `ci_utils/src/paths.rs`.
+# Generation logic is in `src/paths.rs`.
```

Comment:
Actually, both are incorrect. While the latter is where the generated code is included, the generation logic is now in the `build/macros/lib/src/paths.rs`.

Reply:
Updated the comment in ad5353a76a6989ee77be7038788c7dcdeaf35b97